frontend/src/app/accountCreation/LoadingCard/LoadingCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import iconLoader from "../../../../node_modules/uswds/dist/img/loader.svg";
+
+interface Props {
+  message: string;
+}
+
+export const LoadingCard = (props: Props) => {
+  return (
+    <CardBackground>
+      <Card logo bodyKicker={props.message} bodyKickerCentered={true}>
+        <div className="display-flex flex-column flex-align-center">
+          <img className="square-5 chromatic-ignore" src={iconLoader} alt="" />
```

Comment:
An empty alt tag is correct b/c the loading information is already given in the card's message.

Reply:
Hmm, doesn't the alt tag refer to the content of the image though, rather than the card's message? Should the alt be something like `loading spinner`?

Reply:
Yes, it refers to the image. But it's illustrative of the existing text "Verifying security code ..." just above. It can be a bit of a judgment call wrt empty alt text, but the existing text seems sufficient.